InitialState.NET/ISEventDataCollection.cs

```diff
@@ -28,7 +28,7 @@ namespace InitialState.Streaming
     [System.ComponentModel.ListBindable(false)]
     public class ISEventDataCollection : System.Collections.Generic.IList<ISEventData>
     {
-        private List<ISEventData> _events = null;
+        private readonly List<ISEventData> _events = null;

```

Comment:
```suggestion
        private readonly List<ISEventData> _events = 0;
```

Reply:
Hey nlynchjo!  Why =0 and not = null?  I suppose the run-time will make it null automatically at creation so it probably shouldn't even be assigned really...

Reply:
> 

`_events` is `List<T>`. It cant be assigned 0.

Reply:
> 
> 
> Hey nlynchjo! Why =0 and not = null? I suppose the run-time will make it null automatically at creation so it probably shouldn't even be assigned really...

So sorry!! I was supposed to remove this suggestion, this PR was just at the top of the list, and I was demonstrating commit suggestions which is unique to GitHub. I was supposed to remove the suggestion after and I forgot!